nectar/src/database/test_snapshot/v0_1_0.rs

```diff
@@ -0,0 +1,2037 @@
+use conquer_once::Lazy;
+
+pub static TAR: Lazy<Vec<u8>> = Lazy::new(|| {
```

Comment:
Why didn't you just create a DB file with the 0.1.0 tag and check that into source-control? Feels a bit odd to have this embedded here like this.

Reply:
Because in the past, you have expressed that such files in source control were not that clean and hence did PRs to embed the data directly in Rust code.
I just followed your lead here.